watertap/examples/flowsheets/case_studies/electroNP/electroNP_flowsheet_extended.py

```diff
@@ -0,0 +1,496 @@
+    m.fs.feed.conc_mass_comp[0, "X_H"].fix(30 * pyo.units.g / pyo.units.m**3)
+    m.fs.feed.conc_mass_comp[0, "X_PAO"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.feed.conc_mass_comp[0, "X_PP"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.feed.conc_mass_comp[0, "X_PHA"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.feed.conc_mass_comp[0, "X_AUT"].fix(1e-6 * pyo.units.g / pyo.units.m**3)
+    m.fs.feed.properties[0].conc_mass_comp["S_IC"].fix(
+        0.07899 * pyunits.kg / pyunits.m**3
+    )
+    m.fs.feed.properties[0].conc_mass_comp["S_K"].fix(
+        1e-6 * pyo.units.g / pyo.units.m**3
+    )
+    m.fs.feed.properties[0].conc_mass_comp["S_Mg"].fix(
+        1e-6 * pyo.units.g / pyo.units.m**3
+    )
+
+    # Reactor sizing
+    m.fs.R1.volume.fix(1000 * pyo.units.m**3)
+    m.fs.R2.volume.fix(1000 * pyo.units.m**3)
+    m.fs.R3.volume.fix(1333 * pyo.units.m**3)
+    m.fs.R4.volume.fix(1333 * pyo.units.m**3)
+    m.fs.R5.volume.fix(1333 * pyo.units.m**3)
+
+    # Injection rates to Reactions 3, 4 and 5
+    for j in m.fs.props_ASM2D.component_list:
+        if j != "S_O2":
+            # All components except S_O have no injection
+            m.fs.R3.injection[:, :, j].fix(0)
+            m.fs.R4.injection[:, :, j].fix(0)
+            m.fs.R5.injection[:, :, j].fix(0)
+    # Then set injections rates for O2
+    m.fs.R3.outlet.conc_mass_comp[:, "S_O2"].fix(2.72e-3)
+    m.fs.R4.outlet.conc_mass_comp[:, "S_O2"].fix(2.43e-3)
+    m.fs.R5.outlet.conc_mass_comp[:, "S_O2"].fix(4.49e-4)
+
+    # Set fraction of outflow from reactor 5 that goes to recycle
+    m.fs.SP5.split_fraction[:, "underflow"].fix(0.1)
+
+    # Clarifier
+    # TODO: Update once more detailed model available
+    m.fs.CL1.split_fraction[0, "effluent", "H2O"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_A"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_F"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_I"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_N2"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_NH4"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_NO3"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_O2"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_PO4"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_IC"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_K"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "S_Mg"].fix(0.49986)
+    m.fs.CL1.split_fraction[0, "effluent", "X_AUT"].fix(0.022117)
+    m.fs.CL1.split_fraction[0, "effluent", "X_H"].fix(0.021922)
+    m.fs.CL1.split_fraction[0, "effluent", "X_I"].fix(0.021715)
+    m.fs.CL1.split_fraction[0, "effluent", "X_PAO"].fix(0.022)
+    m.fs.CL1.split_fraction[0, "effluent", "X_PHA"].fix(0.02147)
+    m.fs.CL1.split_fraction[0, "effluent", "X_PP"].fix(0.02144)
+    m.fs.CL1.split_fraction[0, "effluent", "X_S"].fix(0.02221)
```

Comment:
Any reason for the slightly different values for split fraction here? I thought they should all be identical for S and X components, respectively.

Reply:
Not so sure, but I followed the same split fraction in `ASM2D_flowsheet_noPHA.py`